lib/util.js

```diff
@@ -478,9 +478,23 @@ function formatValue(ctx, value, recurseTimes) {
   if (ctx.showHidden) {
     keys = Object.getOwnPropertyNames(value);
   } else {
-    keys = Object.keys(value);
-    if (symbols.length !== 0)
-      symbols = symbols.filter((key) => propertyIsEnumerable.call(value, key));
+    // this might throw if `value` is a Module Namespace Object from an
+    // unevaluated module, but we don't want to perform the actual type
+    // check because it's expensive.
+    try {
+      keys = Object.keys(value);
+    } catch (err) {
+      if (err instanceof ReferenceError) {
```

Comment:
Are there other possibilities for this to throw a `ReferenceError`?

Reply:
not that I know of